net/nut/files/nut-sendmail-notify

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+(cat - | /usr/sbin/sendmail root ) <<EOF
+From: root
+To: root
+Subject: UPS $NOTIFYTYPE Notification
+
+$1
+.
+EOF
```

Comment:
On Mon, 2016-07-04 at 06:14 -0700, Yousong Zhou wrote:
[various fixes]

Thank you for your input; that's the kind of thing testing doesn't
really catch but is good to do right.

I'll update the things you pointed out.

Regards,

Daniel